### PR TITLE
Fix testRuns starting collision (fix #743). Bump version

### LINKED ISSUE #743: All tests in a fixture are run at the same time
Reproduced in v0.3.0 and v0.2.3. Works fine in v0.2.1.

```js
var wait = ms => {
    return new Promise(resolve => setTimeout(resolve, ms));
};

fixture `Fixture 1`;

test('Test1', async () => {
    console.log('\ntest1');
    await wait(3000);
});

test('Test2', async () => {
    console.log('\ntest2');
    await wait(4000);
});

test('Test3', async () => {
    console.log('\ntest3');
    await wait(4000);
});
```

Report:
```

test1

test2

test3
 Running tests in:
 - Chrome 52.0.2743 / Windows 7 0.0.0

 Fixture 1
 √ Test1
 √ Test2
 √ Test3


 3 passed (4s)
```

But should be like:
Report:
```
 Running tests in:
 - Chrome 52.0.2743 / Windows 7 0.0.0

 Fixture 1
test1
 √ Test1
test2
 √ Test2
test3
 √ Test3

 3 passed (4s)
```

### PR DESCRIPTION
**Pull request by AlexanderMoskovkin**
/cc @inikulin @AndreyBelym

### CODE REVIEW

**Comment by AlexanderMoskovkin** [comment]
@testcafe-build-bot \restart

**Comment by AlexanderMoskovkin** [comment]
@testcafe-build-bot \retest

File: test/functional/fixtures/regression/gh-743/testcafe-fixtures/two-tests-in-a-fixture.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+var wait = ms => new Promise(resolve => setTimeout(resolve, ms));
+
+test('First test', async () => {
+    await wait(100);
+    expect(secondTestInProgress).to.be.false;
+});
+
+test('Second test', async () => {
+    secondTestInProgress = true;
```
Comment (inikulin) [inline test/functional/fixtures/regression/gh-743/testcafe-fixtures/two-tests-in-a-fixture.js]: Maybe better `secondStarted`? And don't reset it in `Second test`?

**Comment by inikulin** [comment]
\r-

File: test/functional/fixtures/regression/gh-743/testcafe-fixtures/two-tests-in-a-fixture.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+var wait = ms => new Promise(resolve => setTimeout(resolve, ms));
+
+test('First test', async () => {
+    await wait(100);
+    expect(secondStarted).to.be.false;
+});
+
+test('Second test', async () => {
+    secondStarted = true;
+    await wait(500);
```
Comment (inikulin) [inline test/functional/fixtures/regression/gh-743/testcafe-fixtures/two-tests-in-a-fixture.js]: Why we need this wait here? Also, why not `t.wait`?

**Comment by AlexanderMoskovkin** [comment]
@testcafe-build-bot \retest

**Comment by AlexanderMoskovkin** [comment]
FPR

/cc @VasilyStrelyaev please check strings

**Comment by inikulin** [comment]
lgtm

**Comment by VasilyStrelyaev** [comment]
lgtm

**Comment by inikulin** [comment]
@AlexanderMoskovkin Can you publish it, please?

**Comment by AlexanderMoskovkin** [comment]
yep

**Pull request merged**